docs/source/API/core/view/create_mirror.rst

```diff
@@ -71,7 +71,7 @@ Description
 
    - ``ImplMirrorType``: an implementation defined specialization of ``Kokkos::View``.
 
-.. cppkokkos:function:: template <class ViewType, class... ViewCtorArgs> auto create_mirror(Impl::ViewCtorProp<ViewCtorArgs...> const& arg_prop, ViewType const& v);
+.. cppkokkos:function:: template <class ViewType, class... ViewCtorArgs> auto create_mirror(ALLOC_PROP const& arg_prop, ViewType const& v);
```

Comment:
ViewCtorArgs are not a thing anymore, could maybe just do template<class ViewType, class ALLOC_PROP>